src/pages/core-concepts/project-structure.md

```diff
@@ -9,7 +9,8 @@ Astro includes an opinionated folder layout for your project. Every Astro projec
 - `public/*` - Your non-code assets (fonts, icons, etc.)
 - `package.json` - A project manifest.
 
-The easiest way to set up your new project is with `npm init astro`. Check out our [Installation Guide](/quick-start) for a walkthrough of how to set up your project automatically (with `npm init astro`) or manually.
+The easiest way to set up your new project is with `npm init astro`. Check out our [Installation Guide](/quick-start) for a walk-through of how to set up your project automatically (with `npm init astro`) or manually.
```

Comment:
"walkthrough" is a double barrell word, "walk-through"

Reply:
LGTM! 🎉